tests/tests-core/src/main/java/com/gentics/mesh/core/field/node/NodeFieldEndpointTest.java

```diff
@@ -79,17 +79,22 @@ public void testUpdateNodeFieldWithField() {
 		HibNode node = folder("2015");
 		List<HibNode> targetNodes = Arrays.asList(folder("news"), folder("deals"));
 		for (int i = 0; i < 20; i++) {
-			try (Tx tx = tx()) {
-				HibNodeFieldContainer container = boot().contentDao().getFieldContainer(node, "en");
-				HibNode oldValue = getNodeValue(container, FIELD_NAME);
+			HibNode newValue = targetNodes.get(i % 2);
 
-				HibNode newValue = targetNodes.get(i % 2);
+			HibNode oldValue = null;
+			HibNodeFieldContainer container = null;
 
-				// Update the field to point to new target
-				NodeResponse response = updateNode(FIELD_NAME, new NodeFieldImpl().setUuid(newValue.getUuid()));
-				NodeResponse field = response.getFields().getNodeFieldExpanded(FIELD_NAME);
-				assertThat(field.getUuid()).as("New Value").isEqualTo(newValue.getUuid());
+			try (Tx tx = tx()) {					
+				container = boot().contentDao().getFieldContainer(node, "en");
+				oldValue = getNodeValue(container, FIELD_NAME);
+			}
+
+			// Update the field to point to new target
+			NodeResponse response = updateNode(FIELD_NAME, new NodeFieldImpl().setUuid(newValue.getUuid()));
```

Comment:
REST API calls should get removed from the transaction, because of a deadlock at some databases.